cowprotocol/services
# Add driver submission address to the autopilot

# Description
This PR should solve: https://github.com/cowprotocol/services/issues/3045 and https://github.com/cowprotocol/services/issues/2780

# Changes
- Add driver submission address to the autopilot
- Do not request `/solve` to drivers which have been deny listed
- Filter the results by the driver submission address

## How to test
1. Regression test (e2e modified to use the new config)
2. e2e test to check the filtering works

## Related Issues

Fixes #3045
Fixes #2780
